.env.example

```diff
@@ -47,6 +47,9 @@ FORCE_SSL=false
 # You can see its use in user.rb.  PLEASE CHANGE THIS!
 INVITATION_CODE=try-huginn
 
+# If you don't want to require users to have an invitation code, set this to true
```

Comment:
"If you don't want to require new users to have an invitation code in order to sign up, set this to true."